.github/workflows/docker multi-stage build.yaml

```
@@ -0,0 +1,61 @@
+            CONTAINER_ID=$(sudo docker ps -q --filter "publish=8080-8080")
+            if [ ! -z "$CONTAINER_ID" ]; then
+            sudo docker stop $CONTAINER_ID
+            sudo docker rm $CONTAINER_ID
+            fi
+            
+            sudo docker pull ${{ vars.DOCKERHUB_USERNAME }}/${{ vars.DOCKER_IMAGE_TAG_NAME }}:latest
+            sudo docker run -d -p 8080:8080 \
+            -e DB_USERNAME=${{secrets.DB_USERNAME}} \
+            -e DB_PASSWORD=${{secrets.DB_PASSWORD}} \
+            -e DB_URL=${{secrets.DB_URL}} \
+            ${{ vars.DOCKERHUB_USERNAME }}/test:latest
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)